src/store/Store.js

```diff
@@ -95,9 +96,9 @@ class Store {
         this.extensionId,
         {
           type: DISPATCH_TYPE,
+          port: this.portName,
```

Comment:
I think we should call this key `portName` rather than `port`. It's more descriptive on what this actually is (especially because `port` is a reference to a full object in other parts of the app)